ui/admin/package.json

```diff
@@ -89,6 +90,7 @@
     "eslint-plugin-qunit": "^8.0.1",
     "js-bexpr": "hashicorp/js-bexpr#9b4a4b54d85eba67fdfc0990133d1518d890b1e1",
     "loader.js": "^4.7.0",
+    "postcss": "^8.4.31",
```

Comment:
None blocking comment: Why do we need to add postcss now as dependency?

Reply:
I get warning during installation if I don't provide this as a devDependency
`admin@workspace:ui/admin doesn't provide postcss (p389e8), requested by stylelint-config-standard-scss`